exercises/meetup/.meta/DONT-TEST-STUB

```diff
@@ -1,2 +0,0 @@
-Can't build because the parameters of meetupDay are unused
```

Comment:
also, this particular line (parameters) was back when I was trying to build --pedantic (it did not work out)